client/me/help/help-contact/index.jsx

```diff
@@ -410,14 +410,14 @@ class HelpContact extends React.Component {
 		//    requests are sent to the language specific forums (for popular languages)
 		//    we don't tell the user that support is only offered in English.
 		const showHelpLanguagePrompt =
-			config( 'support_locales' ).indexOf( currentUserLocale ) === -1 &&
+			config( 'happychat_support_locales' ).indexOf( currentUserLocale ) === -1 &&
```

Comment:
Although we've been using `happychat` in many places, I'm wondering if we should call it `livechat_support_locales` instead to be less product-specific. cc @Automattic/lighthouse  here for more opinions :)

Reply:
Good point — I'd tend to agree that it's better to use the term `livechat` instead of `happychat` for things that are not specific to Happychat as a product.